survey-data-dictionary.md

```diff
@@ -64,34 +110,48 @@ what each question can have as an answer.
 
 ### About how much month do you expect to earn per year at your first developer job (in US Dollars)?
 
```

Comment:
Typo
How much **MONEY** do you expect to earn per year at your first developer job (in US Dollars)?